tests/testthat/test-infection-integration.R

```diff
@@ -531,3 +560,195 @@ test_that('update_severe_disease renders with no infections', {
     timestep
   )
 })
+
+test_that('calculate_treated returns empty Bitset when there is no clinical treatment coverage', {
+  parameters <- get_parameters()
+  parameters <- set_drugs(parameters = parameters, drugs = list(AL_params))
+  parameters <- set_clinical_treatment(parameters = parameters, drug = 1, timesteps = 1, coverages = 0)
+  parameters <- set_antimalarial_resistance(parameters = parameters,
+                                            drug = 1,
+                                            timesteps = 1,
+                                            artemisinin_resistance = 0.5,
+                                            partner_drug_resistance = 0,
+                                            slow_parasite_clearance_prob = 0,
+                                            early_treatment_failure_prob = 0.2,
+                                            late_clinical_failure_prob = 0,
+                                            late_parasitological_prob = 0,
+                                            reinfection_prob = 0)
+  clinical_infections <- individual::Bitset$new(20)$insert(1:20)
+  timestep <- 5
+  events <- create_events(parameters)
+  variables <- list(
+    state = list(queue_update = mockery::mock()),
+    infectivity = list(queue_update = mockery::mock()),
+    drug = list(queue_update = mockery::mock()),
+    drug_time = list(queue_update = mockery::mock())
+  )
+  renderer <- individual::Render$new(timesteps = 10)
+  
+  treated <- calculate_treated(variables = variables,
+                               clinical_infections = clinical_infections,
+                               parameters = parameters,
+                               timestep = timestep,
+                               renderer = renderer)
+  
+  expect_identical(object = treated$size(), expected = 0, info = "Error: calculate_treated() returning non-zero number of treated individuals
+                 in the absence of clinical treatment")
+  expect_identical(object = treated$to_vector(), expected = numeric(0), info = "Error: calculate_treated() returning non-zero number of treated individuals
+                 in the absence of clinical treatment")
```

Comment:
It seems confusing to check both of these things when the first check is enough

```suggestion
```

Reply:
I've removed the to_vector() test from test_that('calculate_treated returns empty Bitset when there is no clinical treatment coverage' ...) and kept the following:

expect_identical(object = treated$size(), expected = 0, info = "Error: calculate_treated() returning non-zero number of treated individuals in the absence of clinical treatment")